scripts/arnie-quotes.coffee

```diff
@@ -30,10 +30,10 @@ arnie_quotes = [
 module.exports = (robot) ->
     robot.hear /(^|\s)arnie(\s|$|[\W])/ig, (msg) ->
         val = msg.random odds
-        if val > 50
+        if val > 0 # Set Arnie to reply 100% for now
             msg.send msg.random arnie_quotes
 
     robot.hear /(^|\s)arnold(\s|$|[\W])/ig, (msg) ->
         val = msg.random odds
-        if val > 50
+        if val > 0 # Set Arnie to reply 100% for now
```

Comment:
Two small things. 

Since we're not concerned about percentages anymore, you can remove these two preceding lines and always call `msg.send msg.random arnie_quotes`.

I believe you can combine your two `robot.hear` calls into one by adding `(arnie|arnold)` to the regex like so: `/(^|\s)(arnie|arnold)(\s|$|[\W])/ig`